tests/1007/in.def

```diff
@@ -0,0 +1,79 @@
+VERSION 5.8 ;
```

Comment:
How does this file relate to this change?

Reply:
I think that was checked-in by mistake, but I do think it could be useful to keep the def "somewhere" in order to be able to "upgrade" the `in.odb` again when another version of openroad happen to bump the schema.

Or maybe we should keep `.def` file for test inputs as you recently suggested in https://github.com/The-OpenROAD-Project/OpenROAD/pull/2267#issuecomment-1247274980